docs/06/01/setup-guide.md

```diff
@@ -0,0 +1,255 @@
+
+### 2. Create Your Component Adapter Object
+
+Create an object that implements the `ComponentsContextType` interface ([View interface on GitHub](https://github.com/Gusto/embedded-react-sdk/blob/main/src/contexts/ComponentAdapter/useComponentContext.ts)) with your custom components:
+
+```tsx
+import type { ComponentsContextType } from '@gusto/embedded-react-sdk'
+
+const myCustomComponents: ComponentsContextType = {
+  Alert: props => <MyCustomAlert {...props} />,
+  Badge: props => <MyCustomBadge {...props} />,
+  Button: props => <MyCustomButton {...props} />,
+  ButtonIcon: props => <MyCustomButtonIcon {...props} />,
+  CalendarPreview: props => <MyCustomCalendarPreview {...props} />,
+  Card: props => <MyCustomCard {...props} />,
+  Checkbox: props => <MyCustomCheckbox {...props} />,
+  CheckboxGroup: props => <MyCustomCheckboxGroup {...props} />,
+  ComboBox: props => <MyCustomComboBox {...props} />,
+  DatePicker: props => <MyCustomDatePicker {...props} />,
+  Heading: props => <MyCustomHeading {...props} />,
+  Link: props => <MyCustomLink {...props} />,
+  Menu: props => <MyCustomMenu {...props} />,
+  NumberInput: props => <MyCustomNumberInput {...props} />,
+  OrderedList: props => <MyCustomOrderedList {...props} />,
+  ProgressBar: props => <MyCustomProgressBar {...props} />,
+  Radio: props => <MyCustomRadio {...props} />,
+  RadioGroup: props => <MyCustomRadioGroup {...props} />,
+  Select: props => <MyCustomSelect {...props} />,
+  Switch: props => <MyCustomSwitch {...props} />,
+  Table: props => <MyCustomTable {...props} />,
+  Text: props => <MyCustomText {...props} />,
+  TextInput: props => <MyCustomTextInput {...props} />,
+  UnorderedList: props => <MyCustomUnorderedList {...props} />,
+}
+```
+
+### 3. Use GustoProviderCustomUIAdapter
+
+Instead of using the standard `GustoApiProvider`, use the `GustoProviderCustomUIAdapter` ([View on GitHub](https://github.com/Gusto/embedded-react-sdk/blob/main/src/contexts/GustoProvider/GustoProviderCustomUIAdapter.tsx)) to provide your custom components:
+
+```tsx
+import { GustoProviderCustomUIAdapter } from '@gusto/embedded-react-sdk'
+
+function App() {
+  return (
+    <GustoProviderCustomUIAdapter
+      config={{ baseUrl: '/api/gusto/' }}
+      components={myCustomComponents}
+    >
+      {/* Your application components */}
+      <EmployeeOnboardingFlow companyId="company_123" />
+    </GustoProviderCustomUIAdapter>
+  )
+}
+```
+
+### 4. Implement Only the Components You Need
+
+You don't need to implement every component in the `ComponentsContextType` interface. You can choose to implement only the ones you want to customize and let the default components handle the rest by importing the `defaultComponents` ([View on GitHub](https://github.com/Gusto/embedded-react-sdk/blob/main/src/contexts/ComponentAdapter/adapters/defaultComponentAdapter.tsx)):
+
```

Comment:
So for this case we should emphasize that they can just use `GustoProvider` and supply the `components` prop. That's already importing defaultComponents under the hood

Reply:
Agreed, see latest update